Cargo.toml

```diff
@@ -27,15 +27,17 @@ members = [
 edition = "2021"
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/KDAB/cxx-qt/"
-version = "0.5.0"
+version = "0.5.1"
 
+# Note a version needs to be specified on dependencies of packages
+# we publish, otherwise crates.io complains as it doesn't know the version.
 [workspace.dependencies]
 cxx-qt = { path = "crates/cxx-qt" }
 cxx-qt-build = { path = "crates/cxx-qt-build" }
-cxx-qt-gen = { path = "crates/cxx-qt-gen" }
+cxx-qt-gen = { path = "crates/cxx-qt-gen", version = "0.5" }
 cxx-qt-lib = { path = "crates/cxx-qt-lib" }
-cxx-qt-lib-headers = { path = "crates/cxx-qt-lib-headers" }
-qt-build-utils = { path = "crates/qt-build-utils" }
+cxx-qt-lib-headers = { path = "crates/cxx-qt-lib-headers", version = "0.5" }
+qt-build-utils = { path = "crates/qt-build-utils", version = "0.5" }
```

Comment:
wonder if we should bump these versions as well with microreleases. `cargo update` should sort things out, but then this would ensure if you put `0.5.1` as the version of the main crate and dependencies are `0.5.1` as well

Reply:
I think it doesn't matter much, but it would certainly be more consistent.
Might be good for reducing risk of confusion, why one crate has version 0.5.1 and the other 0.5.

I also found this old cargo issue with adding support for referencing other versions, but it was closed as won't fix unfortunately: https://github.com/rust-lang/cargo/issues/4221

Reply:
Changed to be `0.5.1` for the depends to be explicit.